lib/pages/image_viewer/image_viewer_view.dart

```diff
@@ -45,17 +46,15 @@ class ImageViewerView extends StatelessWidget {
                           filterQuality: FilterQuality.none,
                         )
                       : FutureBuilder(
-                          future: controller.widget.event
-                              .downloadAndDecryptAttachment(
-                            // FIXME: change to false after https://github.com/linagora/twake-on-matrix/issues/746
-                            getThumbnail: true,
+                          future: controller.widget.event.getFileInfo(
+                            getThumbnail: false,
```

Comment:
why not `get thumbnail`?

Reply:
i think when open image in full screen, we should download the root file not thumbnail one

Reply:
agree with @sherlockvn